tests/experimental/starry/rotation_test.py

```diff
@@ -0,0 +1,161 @@
+import numpy as np
+import pytest
+from jax import config
+from jaxoplanet._src.experimental.starry.rotation import R
+
+config.update("jax_enable_x64", True)
```

Comment:
We don't want to add this here! We run the tests with and without x64 enabled because we want to support both approaches. If one of the tests will fail at single precision, we can add a conditional skip to it, but it's better to test both cases if we can.

Reply:
I guess we need to adapt the tolerance of `np.testing.assert_allclose` for single precision tests. Should I write two set of separate tests? (did not see any example of that being done in current tests)

Reply:
It already exists here: https://github.com/exoplanet-dev/jaxoplanet/blob/ae01ec962741d1ef08f1b296987ca5dcf8da24a7/src/jaxoplanet/test_utils.py#L5-L11